R/create.cluster.heatmap.R

```diff
@@ -44,14 +44,30 @@ create.cluster.heatmap <- function(
         arr[arr > ccf.limits[2]] <- ccf.limits[2];
         }
 
-    hm <- create.ccf.heatmap(
-        x = arr,
-        cluster.dimensions = 'none',
-        xlab.label = '',
-        xaxis.lab = xaxis.label,
-        colour.scheme = colour.scheme,
-        ...
-        );
+    if (!is.matrix(arr)) {
+        arr <- as.data.frame(arr)
+        colnames(arr) <- as.character(unique(DF$ID))
+
+        hm <- create.ccf.heatmap(
```

Comment:
This is somewhat redundant I think, especially since users can pass through the `same.as.matrix` and `yat` arguments to BPGs `create.heatmap`.